client/src/components/BizCard_Map/sampledata2.json

```diff
@@ -0,0 +1,384 @@
+[
+    {
+      "id": 1,
+      "name": "Ebi-chan",
+      "address_street": "1-8-1 Shimomeguro",
+      "address_city": "Meguro-ku",
+      "address_zip": "153-0064",
+      "phone": "03-5434-3977",
+      "business_type": "izakaya",
+      "capacity": "1-10",
+      "price": 20000,
+      "location":[{
+        "lat":"35.658022",
+        "lng":"139.699277"
+      }],
```

Comment:
it should just be two columns "lat" and "lng" since it will be columns in the `businesses` table